doc/nrf/introduction.rst

```diff
@@ -60,12 +61,33 @@ All of them have a role in the creation of an application, from configuring the
 West
 ====
 
-The Zephyr project includes a tool called west that enables you to manage multiple repositories.
+The Zephyr project includes a tool called west.
+The |NCS| uses :ref:`west <zephyr:west>` to manage the combination of multiple Git repositories and versions.
+Some of west’s features are similar to those provided by Git Submodules and Google’s Repo tool.
+But west also includes custom features required by the Zephyr project that were not sufficiently supported by the existing tools.
+For more details about the reasons behind the introduction of west, see the :ref:`zephyr:west-history` section of the Zephyr documentation.
+
+West's workspace contains exactly one :ref:`manifest repository <zephyr:west-basics>`, which is a main Git repository containing a `west manifest file`_.
+Additional Git repositories in the workspace managed by west are called projects.
+The manifest repository controls which commits to use from the different projects through the manifest file.
+In the |NCS|, the main repository `sdk-nrf`_ contains a west manifest file :file:`west.yml`, that determines the revision of all other repositories.
+This means that sdk-nrf acts as the manifest repository, while the other repositories are projects.
+
 When developing in the |NCS|, your application will use libraries and features from folders that are cloned from different repositories or projects.
 The west tool keeps control of which commits to use from the different projects.
 It also makes it fairly simple to add and remove modules.
 
-A west workspace contains one manifest repository and multiple projects, where the manifest repository controls which commits to use from the different projects.
-For more information, see the :ref:`zephyr:west` user guide.
+Some west commands are related to Git commands with the same name, but operate on the entire west workspace.
+Some west commands take projects as arguments.
+The two most important workspace-related commands in west are ``west init`` and ``west update``.
+The ``west init`` command creates a west workspace, and you typically need to run it only once to initialize west with the revision of the |NCS| that you want to check out.
```

Comment:
This is a very long paragraph. I think you could split it a bit. Maybe a paragraph for init and another for update. Maybe also have the references in a separate paragraph.